src/pages/Contact.js

```diff
@@ -130,6 +130,8 @@ function Contact(){
       </Row>
       <hr />
       <Row>
+    <h2>Send us a message</h2>
```

Comment:
Could look better if you center this.

Reply:
Greetings @padmajabhol, I have done it as per your requirements. Kindly review it. Thank you

Reply:
Greetings @padmajabhol, I have done it as per your requirements. Kindly review it. Thank you

Reply:
I deleted a comment by mistake, sorry about that 😅

Reply:
Is this PR good?

Reply:
No, this should be good to go, thanks!